go/tools/builders/nogo_main.go

```diff
@@ -211,6 +260,11 @@ func checkPackage(analyzers []*analysis.Analyzer, packagePath string, packageFil
 	return diagnostics, facts, nil
 }
 
+type Range struct {
+	from token.Position
+	to   int
+}
```

Comment:
I thought we only care about line numbers here?

It's unclear to me in what scenarios would `from.Filename` and `from.Column` would be helpful. It would be nice to have those scenarios documented as a test.

Reply:
The Filename is because the ranges are built for the entire package. The column helps me associate a comment that applies to the immediately following block. I suppose, in theory, an inline comment could have the same column as a node on the next line, causing the range expand. I think golangci-lint as the same issue though.